File: Projeto-TCC/ACS-Cadastro-Mobile/app/src/main/java/tcc/acs_cadastro_mobile/subModels/EducationEmployment.java
package tcc.acs_cadastro_mobile.subModels;

import org.json.JSONException;
import org.json.JSONObject;

import java.io.Serializable;

import io.realm.RealmObject;
import tcc.acs_cadastro_mobile.Constants;

public class EducationEmployment extends RealmObject implements Serializable {

    private boolean school;
    private String occupation, education, employment;

    public EducationEmployment() {
        this(false, "", "", "");
    }

    public EducationEmployment(boolean school, String occupation, String education, String employment) {
        this.school = school;
        this.occupation = occupation;
        this.education = education;
        this.employment = employment;
    }

    public boolean isSchool() {
        return school;
    }

    public void setSchool(boolean school) {
        this.school = school;
    }

    public String getOccupation() {
        return occupation;
    }

    public void setOccupation(String occupation) {
        this.occupation = occupation;
    }

    public String getEducation() {
        return education;
    }

    public void setEducation(String education) {
        this.education = education;
    }

    public String getEmployment() {
        return employment;
    }

    public void setEmployment(String employment) {
        this.employment = employment;
    }

    public JSONObject asJson() throws JSONException {
        JSONObject json = new JSONObject();
        json.put(Constants.Citizen.SCHOOL.name(), school);
        json.put(Constants.Citizen.OCCUPATION.name(), occupation);
        json.put(Constants.Citizen.EDUCATION.name(), education);
        json.put(Constants.Citizen.EMPLOYMENT.name(), employment);
        return json;
    }
}

File: Projeto-TCC/ACS-Cadastro-Web/subModels/EducationEmployment.php
<?php

class EducationEmployment
{
    const SCHOOL = "SCHOOL";
    const OCCUPATION = "OCCUPATION";
    const EDUCATION = "EDUCATION";
    const EMPLOYMENT = "EMPLOYMENT";
    const EDUCATION_EMPLOYMENT = "EDUCATION_EMPLOYMENT";

    private $school, $occupation, $education, $employment;

    function __construct($school, $occupation, $education, $employment)
    {
        $this->school = $school;
        $this->occupation = $occupation;
        $this->education = $education;
        $this->employment = $employment;
    }

    public function save(AcsDataBase $db, $query)
    {
        $array = array(":" . self::SCHOOL => $this->school,
            ":" . self::OCCUPATION => $this->occupation,
            ":" . self::EDUCATION => $this->education,
            ":" . self::EMPLOYMENT => $this->employment);
        return $db->insert($query, $array);
    }

    public static function getFromArray(array $array)
    {
        return new EducationEmployment($array[self::SCHOOL], $array[self::OCCUPATION], $array[self::EDUCATION], $array[self::EMPLOYMENT]);
    }

}